src/libs/actions/Policy/Category.ts

```diff
@@ -334,6 +334,66 @@ function renamePolicyCategory(policyID: string, policyCategory: {oldName: string
     API.write(WRITE_COMMANDS.RENAME_WORKSPACE_CATEGORY, parameters, onyxData);
 }
 
+function setPolicyCategoryGLCode(policyID: string, categoryName: string, glCode: string) {
```

Comment:
I think we'll want to call this `update*`

Reply:
sounds good